packages/frontend/src/routes.tsx

```diff
@@ -33,6 +33,19 @@ const routes: RouteObject[] = [
     },
     children: [],
   },
+  {
+    path: Routes.LOGOUT,
+    element: <div>LOGGING OUT...</div>,
+    errorElement: <RootBoundary />, // Send user here whenever error is thrown
+    loader: async () => {
+      console.log('Logging out');
+      document.cookie = 'Return=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+      // check if cookie exists then redirect to /
+      if (document.cookie.indexOf('Return') === -1) {
```

Comment:
What if the cookie isn't removed? What then? Can they get stuck here? Should we offer a button here for manual redirect that activates after a few seconds?

Reply:
This comment doesn't necessarily require code changes, just some food for thought.

Reply:
Added comment about this in the code